packages/core/src/components/logo/readme.md

```diff
@@ -18,8 +18,18 @@ Eventueel kunnen een `label` (applicatienaam, portaalnaam of kruimelpadlocatie)
 
 Een `label` toont de titel of naam van de applicatie waar je je als bezoeker in bevindt.
 
-Een `ribbon` is een soort sticker die overlapt met het woordmerk, waar een maximaal vijf-letterige tekst in kan komen te staan om de status van de pagina aan te duiden. Bijvoorbeeld "beta", "test", "acc", of "pre".
-
+Een `ribbon` is een soort sticker die overlapt met het woordmerk, waar een maximaal vijf-letterige tekst in kan komen te staan om de status van de pagina aan te duiden. 
```

Comment:
Ik denk de maximale limiet van 5 letters de onderstaande tabel 'n beetje tegenspreekt. Ik weet niet wat er precies is afgesproken, maar is het niet gewoon 3 letterse geworden?

Reply:
klopt, ik heb het aangepast en ook de omschrijving bij het `@prop` in de `logo.tsx` aangepast!

Reply:
Super